### ISSUE TITLE
Add ゑゐ and corresponding katakana

### ISSUE THREAD

**Issue by AmetistoFF** [aeri/Nihonoari-App#45]
Yes, they're obsolete, but I still think they could be added for people interesting in that kind of thing. And they're even used sometimes in Japan. Also people today are studying hentaigana and manyogana, more reasons to make this addition worthwhile. Anyway, it's only a suggestion.

**Comment by aeri** [comment]
Thanks for the suggestion, however, Nihonoari is intended for an audience that is just getting started with the basics of the language or who want to emphasize mastering well the syllabaries of everyday use, I think including characters that are obsolete in modern Japanese could cause some confusion. 

With respect to the Hentaigana I understand that the current use is merely artistic or to provide an extra solemnity.

What do you think?

**Comment by AmetistoFF** [comment]
Yes, hentaigana use is merely artistic nowadays or if you've intentions to read classic literature. Thankfully there's app for that one, so no need to mix hentaigana into Nihonoari. As for confusion, that may be true. I think the best option is to make an option to enable/disable obsolete kana with some kind of warning message. Although I think ゑ and ゐ are still used in Japanese branding (Yebisu for example), so people eventually will run into them.

Anyway, it's up to you decide. I can only say that some people like me (who learn useless stuff for the sake of it) are going to appreciate this addition.
